flashcard.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import FlashcardBuild from './flashcardbuild'
+
+export default class Flashcard extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      flashcard: []
+    }
+    this.saveOnClick = this.saveOnClick.bind(this)
+  }
+
+  saveOnClick() {
+    const question = document.getElementById('question').value
```

Comment:
In general, we don't query the DOM directly in React apps. Review the `React with Forms` exercise.